Sources/Extensions/UIKit/UITextFieldExtensions.swift

```diff
@@ -79,14 +79,27 @@ public extension UITextField {
 		self.attributedPlaceholder = NSAttributedString(string: holder, attributes: [NSForegroundColorAttributeName: color])
 	}
   
-  /// SwifterSwift: Add padding to the left of the textfield rect.
-  ///
-  /// - Parameter padding: amount of padding to apply to the left of the textfield rect.
-  public func addPaddingLeft(_ padding: CGFloat) {
+    /// SwifterSwift: Add padding to the left of the textfield rect.
+    ///
+    /// - Parameter padding: amount of padding to apply to the left of the textfield rect.
+    public func addPaddingLeft(_ padding: CGFloat) {
     let paddingView = UIView(frame: CGRect(x: 0, y: 0, width: padding, height: frame.height))
     leftView = paddingView
     leftViewMode = .always
-  }
+    }
+    
+    /// Add padding to the left of the textfield rect.
```

Comment:
@xingfuqiu All extensions need to be marked with SwifterSwift in the description